_includes/nav.html

```diff
@@ -0,0 +1,28 @@
+<nav class="navbar navbar-expand-lg bg-dark" data-bs-theme="dark">
+  <div class="container-fluid mx-3">
+    <a class="navbar-brand flex-shrink-1" href="{{ '/' | absolute_url }}">
+      <img src="{{ 'images/kroxylicious_logo_rgb_hori_reverse.svg' | absolute_url }}"
+           alt="Kroxylicious: Network proxy framework for Apache Kafka"
+           width=112 height=28>
+    </a>
+    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+      <span class="navbar-toggler-icon"></span>
+    </button>
+    <div class="collapse navbar-collapse justify-content-end" id="navbarSupportedContent">
+      <ul class="navbar-nav">
+        <li class="nav-item me-3">
+          <a class="nav-link" href="{{ '/quickstarts/' | absolute_url }}">Quickstarts</a>
+        </li>
+        <li class="nav-item me-3">
+          <a class="nav-link" href="https://github.com/kroxylicious/kroxylicious/releases">Download</a>
+        </li>
+        <li class="nav-item me-3">
+          <a class="nav-link" href="https://github.com/kroxylicious">GitHub</a>
+        </li>
+        <li class="nav-item me-3">
+          <a class="nav-link" href="{{ '/blog/' | absolute_url }}">Blog</a>
+        </li>
+      </ul>
+    </div>
```

Comment:
I found it quite strange to mix internal and external links like this. 

At a minimum I think we should group `Quickstarts` and `Blog` together and first. 

I'm sure we previously talked about targeting three different personas. 

1. Filter Authors
2. Administrators
3. Managers/Decision makers.

I wonder if we should be providing Nav headings targeting each of those?

Reply:
> I found it quite strange to mix internal and external links like this.

Yeah, good point. I just wasn't really sure where else to put links to where to get Kroxylicious, so they ended up there. Perhaps these could live on their own page, or could become icons instead of text links (similar to what [Strimzi](https://strimzi.io/) do with their site nav). I agree that `Quickstarts` and `Blog` should be grouped.

> I wonder if we should be providing Nav headings targeting each of those?

They would all lead to the same page (Quickstarts), so I'm not so sure about separating them in the nav like that. The quickstarts page is currently just placeholder content, but it has a tabbed design that allows for the quickstarts for any number of personas to live on the same page (I've got placeholders for filter authors and admins there currently). We could have a page for "Why should you use Kroxylicious?" that targets the manager persona, but I'm not sure calling people (or groups of people) out directly in the nav is what we want to be doing.

I'm not even sure about using the headings "For Kafka Administrators" and "For Filter Developers" in the quickstarts, I think maybe it could be something more to do with how Kroxylicious will be used instead of who will be using it, like "For Running Kroxylicious" and "For Developing Custom Filters" instead (or some less verbose alternative). I don't know, maybe I'm overthinking it.

Reply:
> Yeah, good point. I just wasn't really sure where else to put links to where to get Kroxylicious, so they ended up there. 

I think having a top right link to github is fairly idomatic these days and we do want to drive traffic to the GitHub repos so I think it qualifies for inclusion in the nav bar. 

>They would all lead to the same page (Quickstarts), so I'm not so sure about separating them in the nav like that

hmm I think what I was meaning was quick starts are just one bit of content for a given persona (and don't exist for all). So I'm questioning if `quckstarts` is the Nav entry or if its really a home for `Developer`, `Administrator` and `Overview` content?